auth0/src/main/java/com/auth0/android/authentication/storage/CryptoUtil.java

```diff
@@ -400,6 +419,9 @@ private void deleteAESKeys() {
             SecretKey key = new SecretKeySpec(getAESKey(), ALGORITHM_AES);
             Cipher cipher = Cipher.getInstance(AES_TRANSFORMATION);
             String encodedIV = storage.retrieveString(KEY_IV_ALIAS);
+            if (TextUtils.isEmpty(encodedIV)) {
```

Comment:
If there's any possibility of the new alias being in use but the IV value being empty (seems unlikely, but then again the comment below implies it's possible), then this could find an old value?

I suppose that would just result in the same result (forced logout), as long as the KEY_IV_ALIAS read succeeded the second time. Might be something to consider if we need rotation.